CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/).
 
+## [0.4.0] - 2024-10-15
+
+### Added
+- **Full Snapshots support** allowing complete data collection in each full snapshot collected from the Collector for improved reporting and analysis.
```

Comment:
Did we really add functionality? This seems like and under-the-hood change from a user perspective.

Reply:
We're collecting more data, but it's not shown in the UI.